src/components/EpisodePreview/EpisodePreview.jsx

```diff
@@ -0,0 +1,120 @@
+import React, { useContext } from "react";
+import {
+	EpisodesContext as Episodes
+} from "COMPONENTS/Episodes";
+
+export default function EpisodePreview(props){
+
+	// CONTEXT
+	//-----------------------------
+	const {
+		episodeCount, 
+		activeIndex,
+		setActiveIndex,
+		overlayActive,
+		setOverlayActive
+	} = useContext(Episodes);
+
+
+	// RENDER
+	// ----------------------------
+	const { 
+		index,
+		number: episodeNo,
+		title,
+		image: {
+			src: thumbSrc,
+			description: thumbDesc
+		},
+		video: {
+			src: videoSrc
+		}
+	} = props;
+
+	const previewId = `episode__${episodeNo}__preview`;
+
+	// handle visibility of active episode and episode preview overlay
+	const isActiveEpisode 	= activeIndex === index;
+	const thisOverlayActive = isActiveEpisode && overlayActive;
+	const hidden 			= !thisOverlayActive;
+
+	const isFirst 	= index === 0;
+	const isLast 	= index === episodeCount - 1;
+
+	const prevEpisodeNo = episodeNo - 1;
+	const nextEpisodeNo = episodeNo + 1;
+
+	function handleNavPressed(e, newIndex){
+		e.preventDefault();
+
+		setActiveIndex(newIndex);
+	}// handleNavPressed
+
+
+	function showOverlay(e, show){
+		e.preventDefault();
+
+		setOverlayActive(show);
+	}// showOverlay
+
+	return (
+		<aside>
+			<a
+				href={ videoSrc }
+				aria-controls={ previewId }
+				aria-label="Watch episode preview."
```

Comment:
I think we can improve this a liiitle more by doing something like `Watch preview for episode 1: The Awakening.`

Reply:
Good shout, done.